datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_mib_filenames.py

```diff
@@ -0,0 +1,148 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import os
+import re
+from collections import namedtuple
+
+import click
+from tabulate import tabulate
+
+from ...console import CONTEXT_SETTINGS, echo_info, echo_warning
+
+OPTION_ALL = '*'
+
+OPTION_HELP = '''Use a supported option to select all files, one single file, a range of files or multiple ranges
+Option supports:
+> * all
+> 1 single index
+> 1-4 range of indexes from 1 to 4 excluded
+> 1-4:6 multiple ranges
+Example:
+1-3:6-9
+selects files 1,2 and 6,7,8
+2
+selects file 2
+1:3:5-9
+selects files 1,3,5,6,7,8
+1:4
+selects files 1,2,3
+'''
+
+# (
+#   (
+#     \d+     # one or more digits
+#     (-\d+)* # zero or one character'-' followed by one or more digits
+#   ){1} # exactly once
+#   (
+#     :\d+    # one character ':' followed by one or more digits
+#     (-\d+)* # zero or one character'-' followed by one or more digits
+#   )*  # one or more
+# )
+# |  # OR
+# \* # one character '*'
+OPTION_PATTERN = r'((\d+(-\d+)*){1}(:\d+(-\d+)*)*)|\*'
+
+MibFile = namedtuple('MibFile', ['index', 'folder', 'base_filename', 'name', 'is_valid'])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Validate MIB file names')
+@click.argument('mib_files', nargs=-1)
+@click.option('--fix-silent', '-s', is_flag=False, help='Rename all invalid MIB files without prompting to confirm')
```

Comment:
Minor thing. I feel the unattended/silent rename all might be more suitable as default since it "usually" (?) what we want, rename all the files to make them work with snmp tools.

Reply:
I took an opinionated decision on this, I let you vote on this @gmarabout @AlexandreYang